src/main/resources/script_templates/ImageJ2-Scripts/Z_Projection.py

```diff
@@ -0,0 +1,54 @@
+# @ImageJ ij
+# @Dataset data
+# @String(label="Projection Type",choices={"Max","Mean","Median","Min", "StdDev", "Sum"}) proj_type
+ 
+ 
+from net.imagej.axis import Axes
+from net.imagej.ops import Ops
+ 
+ 
+def proj_method(method):
+    return {
+        'Max': Ops.Stats.Max,
+        'Mean': Ops.Stats.Mean,
+        'Median': Ops.Stats.Median,
+        'Min': Ops.Stats.Min,
+        'StdDev': Ops.Stats.StdDev,
+        'Sum': Ops.Stats.Sum,
+    }.get(method, Ops.Stats.Max)
+ 
+ 
+def main():
+ 
```

Comment:
In the future we might want to work out a strategy to create "friendly" signatures for ops via scripts.  What I mean is instead of calling this ```main()``` you could just call it ```project(...)``` and then we could import it into other scripts, to use as a concise way to call projections.